emailpassword/docs/common-customizations/sessions/securing-component.md

```diff
@@ -0,0 +1,25 @@
+---
+id: securing-component
+title: Securing component
+hide_title: true
+---
+
+# Securing a component
```

Comment:
This title is very confusing.. something like "Securing a website route" is more clear